src/views/footnotes_element.ts

```diff
@@ -23,6 +24,20 @@ export class FootnotesElementView<
 > extends BlockView<PropsType> {
   public elementType = 'div'
 
+  // The attempt to use this method failed because the footnote element in the table footer was created as a DIV element view instead of using this node view. Consequently, it doesn't trigger this updateContents function.
+
+  // One potential solution is to replace 'createNodeOrElementView' with 'createNodeView'. However, this change will alter the DOM structure of the table footer, potentially introducing additional elements related to block views. Therefore, additional effort will be required to ensure the correct display of the table footer.
+
+  // public updateContents = () => {
```

Comment:
I think as long as we have a corresponding node that is displayable, we should delegate to PM as much handling as possible, so I would say creating a node view would be in the long run a more beneficial solution. However I'm not so sure about sharing a single view for the generic footnotes and table footnotes.

Reply:
I agree, using the node view itself makes more sense. I went with this solution in the end.

Regarding sharing a single view for the regular footnotes and table footnotes, it sounds fine until now, I just had to hide/discard a few things related to the block view as I don't want these for the table footnotes element.